zalando/zalando.go

```diff
@@ -20,9 +20,9 @@ var AccessTuples []AccessTuple
 
 // AccessTuple is the type defined for use in AccessTuples.
 type AccessTuple struct {
-	Realm string // p.e. "employees", "services"
-	Uid   string // UnixName
-	Cn    string // RealName
+	Realm string `yaml:"Realm,omitempty"` // p.e. "employees", "services"
+	UID   string `yaml:"Uid,omitempty"`   // UnixName
```

Comment:
This change will break clients, please do not rename members of a public struct.